project/LienShark/Readme.rst

```diff
@@ -0,0 +1,99 @@
+=================
+Project
+=================
+Efficient-Eigenvalue-Approximation-using-Gersgorin-Theorem-and-PCA-for-Dimensionality-Reduction 
+
+Basic Information
+=================
+
+GitHub repository :https://github.com/LienShark/Efficient-Eigenvalue-Approximation-using-Gersgorin-Theorem-and-PCA-for-Dimensionality-Reduction
+About:I will try to use Gersgorin Theorem to find eigenvalue approximation to do spectral,
```

Comment:
Spectral and principal value analysis are classical numerical methods.  You can tell a lot of stories with them.